iModelJsNodeAddon/api_package/ts/src/test/DgnDb.test.ts

```diff
@@ -510,4 +510,25 @@ describe("basic tests", () => {
       }],
     });
   });
+
+  it("testConvertEC2XmlSchemas", async () => {

```

Comment:
This is not enough of a test.  It doesn't test any actual conversions that are done by the schema converter and does not test any actual dependencies between schemas.  See existing schema converter tests for an example of a schema that actually has something to convert.

Reply:
Added additional tests to verify that schema metadata is also converted